### PR TITLE
064 changelog

### PR DESCRIPTION
**Pull request by kelset**
This is the first draft of the changelog entry for 0.64.

Generated after the release of rc0.

We'll iterate on this and add new commits as we go along.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,398 @@
...
+- Get ripple drawables by id ([c8ed2dbbb2](https://github.com/facebook/react-native/commit/c8ed2dbbb287deed05a8782fb8665c1edf45bbac) by [@vonovak](https://github.com/vonovak))
+- Internal change to make `PlatformConstants` use the same method to determine `ServerHost` as other code paths ([d20ac69fc8](https://github.com/facebook/react-native/commit/d20ac69fc87ef4f9f5921356c6bbfc0587a9375c) by [@ide](https://github.com/ide))
+- Make CatalystInstance.getNativeModule nullable ([1cef72af04](https://github.com/facebook/react-native/commit/1cef72af047bd2e56d774858093f1ee6269c2d27) by [@RSNara](https://github.com/RSNara))
+- ScrollView now supports `contentOffset` ([ed29ba13f9](https://github.com/facebook/react-native/commit/ed29ba13f97f240c91fdf6c0ef3fb601046697b9) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Update Gradle Wrapper to 6.3 ([8988a073b4](https://github.com/facebook/react-native/commit/8988a073b48df0f0cd4a7126edf1a421f4537d58) by [@friederbluemle](https://github.com/friederbluemle))
```
Comment (mikehardy) [inline CHANGELOG.md:133]: Seems like the gradle wrapper and plugin changes could be coalesced, not necessary to mention that during the dev cycle it went to 6.3 then 6.5 then 6.6 etc ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,398 @@
...
+- Deprecate support of Android API levels 16 to 18. The new minSDK version will be 19+ moving forward ([973198667d](https://github.com/facebook/react-native/commit/973198667d7bbbf3b5d8890fc0a53dc99d0bce18) by [@mdvacca](https://github.com/mdvacca))
+
+#### iOS specific
+
+- IOS: Animated image should animate at the same speed regardless of framerate ([b0d0e51a77](https://github.com/facebook/react-native/commit/b0d0e51a7724dcefe3ce1c2dfb334a731b2a385c) by [@p-sun](https://github.com/p-sun))
+
+### Removed
+
+- `Text.viewConfig` is no longer exported. ([06ce643565](https://github.com/facebook/react-native/commit/06ce64356594a921cd9ae4f71c15dd56dd0e53a3) by [@yungsters](https://github.com/yungsters))
+- Added `listenerCount()` to `DeviceEventEmitter` and `NativeEventEmitter`. ([b11d6ecbb8](https://github.com/facebook/react-native/commit/b11d6ecbb8bb2f0d6f423be6775e587f4e9b1c4d) by [@yungsters](https://github.com/yungsters))
```
Comment (mikehardy) [inline CHANGELOG.md:168]: seems strange to have an "Added ..." in the removed section?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,398 @@
 # Changelog
 
+## v0.64.0-rc.0
+
+### Breaking
+
+- Enable `inlineRequires` by default. Gives a performance benefit but slightly different JS execution order ([959365a902](https://github.com/facebook/react-native/commit/959365a90216ee14d0f8b5d2f4653a1ab4c10d7e) by [@GantMan](https://github.com/GantMan))
+- : Pressable: Rename pressRectOffset to pressRetentionOffset to be consistent with other touchables ([66103277d2](https://github.com/facebook/react-native/commit/66103277d289da5ab2c68389b56327316e69dcb3) by [@TheSavior](https://github.com/TheSavior))
```
Comment (donatoaguirre24) [inline CHANGELOG.md:8]: extra colon here 😁

```suggestion
- Pressable: Rename pressRectOffset to pressRetentionOffset to be consistent with other touchables ([66103277d2](https://github.com/facebook/react-native/commit/66103277d289da5ab2c68389b56327316e69dcb3) by [@TheSavior](https://github.com/TheSavior))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,398 @@
...
+- Adds support for the `onProgress` event on `Image` ([fa0e6f8051](https://github.com/facebook/react-native/commit/fa0e6f8051d2208af467b789a2a9306ec7ddad76) by [@yungsters](https://github.com/yungsters))
+- Exposed getFlex method as part of ReactShadowNode API ([6570f7887b](https://github.com/facebook/react-native/commit/6570f7887b8824705ae09b5653d631428e17bc5f) by [@mdvacca](https://github.com/mdvacca))
+- Add \*.hprof files to gitignore ([69ce9c21d4](https://github.com/facebook/react-native/commit/69ce9c21d433a23ffb9934062b46fa64277ee255) by [@enesozturk](https://github.com/enesozturk))
+- Move DevSettingsActivity from main to debug ([d8e6c45782](https://github.com/facebook/react-native/commit/d8e6c45782a5c9132bb7ec315fe0b9ba3999e830) by [@invalid-email-address](https://github.com/invalid-email-address))
+
+#### iOS specific
+
+- Add disableButtonsIndices option to ActionSheetIOS component ([a7c1c5aff2](https://github.com/facebook/react-native/commit/a7c1c5aff24671bba609caeb82092a8de3d3b232) by [@lukewalczak](https://github.com/lukewalczak))
```
Comment (levibuzolic) [inline CHANGELOG.md:54]: There's a semi-duplicate item below which can be combined. Looks like it was caused by a revert.
```suggestion
- Add disableButtonsIndices option to ActionSheetIOS component ([a7c1c5aff2](https://github.com/facebook/react-native/commit/a7c1c5aff24671bba609caeb82092a8de3d3b232) and [f0bf4b0986](https://github.com/facebook/react-native/commit/f0bf4b09863f0426dd5f6753fb75a174c6c74cbd) by [@lukewalczak](https://github.com/lukewalczak))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,398 @@
...
+- This is the first step to build RCTParagraphComponentAccessibilityProvider. The main idea of RCTParagraphComponentAccessibilityProvider is to provide an array of accessible elements for the AttributedString in PCTParagraphComponentView. ([ffa07254de](https://github.com/facebook/react-native/commit/ffa07254de914f7876a17ec2d1ecac1dc10b116a) by [@ZHUANGPP](https://github.com/ZHUANGPP))
+- Allow hotkeys to be used without command key ([f2b9ec7981](https://github.com/facebook/react-native/commit/f2b9ec798172db76dfb55f390e1fcea90dd341da) by [@rickhanlonii](https://github.com/rickhanlonii))
+- Add disableButtonsIndices option to ActionSheetIOS component ([f0bf4b0986](https://github.com/facebook/react-native/commit/f0bf4b09863f0426dd5f6753fb75a174c6c74cbd) by [@lukewalczak](https://github.com/lukewalczak))
```
Comment (levibuzolic) [inline CHANGELOG.md:64]: Remove din favour of a combined item above.
```suggestion
```

**Comment by kelset** [comment]
thanks everyone for the feedback, we are actually going to close this PR in favour of this one: https://github.com/react-native-community/releases/pull/216 since Jeremy has already started addressing some fixes

**Pull request closed without merging**